src/main/java/senla/com/socialnetwork/model/User.java

```diff
@@ -39,6 +42,14 @@ public class User {
     @OneToMany(mappedBy = "user")
     private Set<Comment> comments;
 
+    @ManyToMany
+    @JoinTable(
+            name = "user_role",
+            joinColumns = {@JoinColumn(name = "user_id")},
+            inverseJoinColumns = {@JoinColumn(name = "role_id")}
+    )
+    private Set<Role> roles;
```

Comment:
when a property has a collection type, initialize it when you declare it. Later, you could face NPE otherwise.